tools/travis/travis_linux_before_install.sh

```diff
@@ -1,6 +1,35 @@
 #!/bin/bash
 set -e
 
+if [ "$ARCH" = "freertoslwip" ]; then
+    echo -e "\r\n==Compile multithreaded version==" && echo -en 'travis_fold:start:before_install.build.multithread\\r'
+    mkdir -p build && cd build
+    cmake \
```

Comment:
cmake and make commands should not be executed in the install script, but in the `travis_linux.sh` script.

Then you should be able to build the whole open62541 stack and not ignore the build error

Reply:
Fixed with https://github.com/open62541/open62541/pull/3023/commits/5f31676c03d8fe957e07f31f0cd336b460b4cec9